targets/TARGET_STM/TARGET_STM32F4/TARGET_STM32F407xE/objects.h

```diff
@@ -0,0 +1,67 @@
+/* mbed Microcontroller Library
+ *******************************************************************************
+ * Copyright (c) 2014, STMicroelectronics
```

Comment:
please add SPDX identifier for new files like this one

Reply:
@0xc0170 Thanks. I added SPDX identifiers.